khmer/utils.py

```diff
@@ -249,24 +248,30 @@ def write_record_pair(read1, read2, fileobj):
     write_record(read2, fileobj)
 
 
+def clean_input_reads(screed_iter):
+    for record in screed_iter:
+        record.cleaned_seq = record.sequence.upper().replace('N', 'A')
+        yield record
+
+
 class ReadBundle(object):
     def __init__(self, *raw_records):
         self.reads = [i for i in raw_records if i]
-        self.cleaned_seqs = [r.sequence.replace('N', 'A') for r in self.reads]
```

Comment:
I guess I'm fine with moving the read cleaning code to a function rather than a class. (Having the cleaned seq as part of the original record is better organization anyway, IMO.) But then that leaves the question of what the `ReadBundle` class is really for. Just aggregation? If so, we need to update the docs from my last PR to make sure we're clear about what code is doing what.